src/azul/chalice.py

```diff
@@ -857,14 +857,18 @@ def robots_txt():
         return locals()
 
     def default_specs(self):
+        retry_after = format_description('''
+            When handling this response, clients should wait the number of
+            seconds specified in the `Retry-After` header and then retry the
+            request.
```

Comment:
```suggestion
            When handling this response, clients should honor the `Retry-After` header.
```

Implies the waiting to retry.